packages/notification-center/src/store/novu-theme.context.ts

```diff
@@ -1,10 +1,79 @@
 import React from 'react';
 import { ColorScheme } from '../index';
+import { ICommonTheme } from './novu-theme-provider.context';
 
 export interface IThemeContext {
   colorScheme: ColorScheme;
+  theme: INovuTheme;
+  common: ICommonTheme;
 }
 
 export const ThemeContext = React.createContext<IThemeContext>({
   colorScheme: 'light',
+  theme: null,
+  common: null,
 });
+
+export interface INovuTheme {
+  layout?: IThemeLayout;
+  header?: IThemeHeader;
+  popover?: IThemePopover;
+  notificationListItem?: IThemeNotificationListItem;
+  footer?: IThemeFooter;
+  mainColor?: string;
+  bellGradientDot?: IThemeBellGradientDot;
+}
+
+export interface IThemeLayout {
+  background?: string;
+  boxShadow?: string;
+  borderRadius?: string;
+  wrapper?: {
+    secondaryFontColor?: string;
+  };
+}
+
+export interface IThemeHeader {
+  badgeColor?: string;
+  badgeTextColor?: string;
+  fontColor?: string;
+}
+
+export interface IThemePopover {
+  arrowColor?: string;
+}
+
+export interface IThemeNotificationListItem {
+  seen?: {
+    fontColor?: string;
+    background?: string;
+    timeMarkFontColor?: string;
+  };
+  unseen?: {
+    fontColor?: string;
+    background?: string;
+    boxShadow?: string;
+    notificationItemBeforeBrandColor?: string;
+    timeMarkFontColor?: string;
+  };
+}
+
+export interface IThemeFooter {
+  logoTextColor?: string;
+  logoPrefixFontColor?: string;
+}
+
+export interface IThemeBellGradientDot {
+  color?: {
+    stopColor?: string;
+    stopColorOffset?: string;
+    borderColor?: string;
+  };
```

Comment:
```suggestion
  color?: string | {
    stopColor?: string;
    stopColorOffset?: string;
    borderColor?: string;
  };
```

In this case, we can do simple color or a gradient object.

Reply:
Agree